include/seqan3/alphabet/cigar/all.hpp

```diff
@@ -0,0 +1,43 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
```

Comment:
```suggestion
// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
```

This changes in between, sorry.

Reply:
Also in the other files.